reo/src/load_profile.py

```diff
@@ -368,9 +368,13 @@ def bau_outage_check(critical_loads_kw, existing_pv_kw_list, gen_existing_kw, ge
     :param time_steps_per_hour: int
     :return: bool, int for number of time steps the existing generator and PV can meet the critical load, and
         boolean for if the entire critical load is met
+    ## TODO: add generator_fuel_use_gal 
```

Comment:
delete TODO because it seems done?

Reply:
Addressed in [this commit (919ee48)](https://github.com/NREL/REopt_Lite_API/pull/274/commits/919ee4847468b15d9c049f94d9c94ebb58aadd51)